Work41/Customer.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Work41
+{
+    class Customer
+    {
+        private string CustomerName;
```

Comment:
```suggestion
private string CustomerName { get; set; }
```

Reply:
if the field is private, why should he get / set? get cannot be used because the field is not visible in the mein method, and the value is set through the constructor, so the set control also occurs there